example/src/demos/custom-sorter.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+
+import CountryPhoneInput, {
+  CountryPhoneInputValue,
+  ConfigProvider,
+} from 'antd-country-phone-input';
+import en from 'world_countries_lists/data/countries/en/world.json';
+import 'antd/dist/antd.css';
+import 'antd-country-phone-input/dist/index.css';
+
+const App = () => {
+  const [value, setValue] = useState<CountryPhoneInputValue>({ short: 'US' });
+
+  return (
+    <ConfigProvider
+      locale={en}
+      areaSorter={(a, b) => {
+        if (a.name && b.name) {
+          const nameA = a.name.toUpperCase();
+          const nameB = b.name.toUpperCase();
+          if (nameA < nameB) {
+            return -1;
+          }
+          if (nameA > nameB) {
+            return 1;
+          }
+          return 0;
+        }
+        return 0;
+      }}
+    >
+      <CountryPhoneInput
```

Comment:
It's almost done! A simple line is enough here. No need to add the `className` and blabla.
```jsx
<CountryPhoneInput />
```

Reply:
Like I don't understand?

Reply:
You mean I should remove the className?

Reply:
Oh Seen it

Reply:
Made the change